reana_workflow_engine_serial/tasks.py

```diff
@@ -187,3 +189,10 @@ def run_serial_workflow(workflow_uuid, workflow_workspace,
              'at {workflow_workspace}.'.format(
                  workflow_uuid=workflow_uuid,
                  workflow_workspace=workflow_workspace))
+
+
+@task_revoked.connect
+def on_task_revoked(*args, **kwargs):
```

Comment:
The current approach doesn't completely solve the problem since we do not have the knowledge of which jobs have been submitted to R-J-Controller so we can not stop them. A separate repository [here](https://github.com/diegodelemos/stop-celery-task) with a toy example to solve it.